build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts

```diff
@@ -41,7 +41,7 @@ repositories {
     }
     maven {
         name = "Kotlin EAP repository"
-        url = uri("https://cache-redirector.jetbrains.com/maven.pkg.jetbrains.space/kotlin/p/kotlin/bootstrap")
```

Comment:
Why not leave in the `cache-redirector`? Do you know which one is the suggested repository by the Kotlin team?

Reply:
Where did you get `cache-redirector`? I get the url here: https://mvnrepository.com/repos/space-kotlin-dev

Reply:
So it seems the `https://maven.pkg.jetbrains.space/` version is used by documentation here: https://github.com/JetBrains/compose-jb/blob/master/tutorials/Getting_Started/README.md#create-new-compose-project-without-the-wizard

Reply:
Good, then let's go with your version!